user-service/src/main/java/com/movie/recommender/user/model/dto/UserDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.movie.recommender.user.model.dto;
+
+public class UserDTO {
+    private String username;
+    private String password;
+
+    public UserDTO() {}
+
+    public UserDTO(String username, String password) {
```

Comment:
add `@Data, @AllArgsConstructor, and @NoArgsConstructor` annotations
these annotations will automatically generate all necessary getters, setters, and constructors

additionally, these annotations are already used in `location service`, so using them here will help maintain a uniform style across the project